proposals/extensions.md

```diff
@@ -50,6 +50,38 @@ receiver_parameter // add
 Extension declarations shall only be declared in non-generic, non-nested static classes.  
 It is an error for a type to be named `extension`.  
 
+### Scoping rules
+
+The type parameters and receiver parameter of an extension declaration are in scope within the body of the extension declaration. It is an error to refer to the receiver parameter from within a static member. It is an error for members to declare type parameters or parameters with the same name as a type parameter or receiver parameter of the extension declaration.
```

Comment:
>It is an error for members to declare type parameters or parameters with the same name as a type parameter or receiver parameter of the extension declaration. [](http://example.com/codeflow?start=209&length=158)

I think it is worth mentioning that the same goes for local variables and local function declared directly by members (not inside lamndas or local functions)